Formula/kapacitor.rb

```diff
@@ -20,8 +20,21 @@ class Kapacitor < Formula
   end
 
   depends_on "go" => :build
+  depends_on "rust" => :build
+
+  # NOTE: The version here is specified in the go.mod of influxdb.
```

Comment:
Is this comment right? It refers to `influxdb`. Do we need to check that `go.mod` too? Or is it in `kapacitor`'s `go.mod`?

Reply:
good catch, fixing the comment reference now. :)

Reply:
Thanks!